proxy/envoy/testdata/envoy-ssl-context.json.golden

```diff
@@ -0,0 +1,297 @@
+            "filters": [
+              {
+                "type": "decoder",
+                "name": "mixer",
+                "config": {
+                  "mixer_server": "mixer:9091",
+                  "mixer_attributes": {
+                    "target.service": "hello.default.svc.cluster.local",
+                    "target.uid": "10.1.1.0"
+                  },
+                  "forward_attributes": {
+                    "source.service": "hello.default.svc.cluster.local",
+                    "source.uid": "10.1.1.0"
+                  }
+                }
+              },
+              {
+                "type": "decoder",
+                "name": "router",
+                "config": {}
+              }
+            ],
+            "access_log": [
+              {
+                "path": "/dev/stdout"
+              }
+            ]
+          }
+        }
+      ],
+      "bind_to_port": false
+    },
+    {
+      "port": 1090,
+      "filters": [
+        {
+          "type": "read",
+          "name": "tcp_proxy",
+          "config": {
+            "stat_prefix": "tcp",
+            "route_config": {
+              "routes": [
+                {
+                  "cluster": "in:1090",
+                  "destination_ip_list": [
+                    "10.1.1.0/32"
+                  ]
+                }
+              ]
+            }
+          }
+        }
+      ],
+      "bind_to_port": false
+    },
+    {
+      "port": 5001,
+      "filters": [],
+      "bind_to_port": true,
+      "use_original_dst": true
+    }
+  ],
+  "admin": {
+    "access_log_path": "/dev/stdout",
+    "port": 5000
+  },
+  "cluster_manager": {
+    "clusters": [
+      {
+        "name": "in:1081",
+        "connect_timeout_ms": 1000,
+        "type": "static",
+        "lb_type": "round_robin",
+        "hosts": [
+          {
+            "url": "tcp://127.0.0.1:1081"
+          }
+        ]
+      },
+      {
+        "name": "in:1090",
+        "connect_timeout_ms": 1000,
+        "type": "static",
+        "lb_type": "round_robin",
+        "hosts": [
+          {
+            "url": "tcp://127.0.0.1:1090"
+          }
+        ]
+      },
+      {
+        "name": "in:80",
+        "connect_timeout_ms": 1000,
+        "type": "static",
+        "lb_type": "round_robin",
+        "hosts": [
+          {
+            "url": "tcp://127.0.0.1:80"
+          }
+        ]
+      },
+      {
+        "name": "out:world.default.svc.cluster.local:custom",
+        "service_name": "world.default.svc.cluster.local:custom",
+        "connect_timeout_ms": 1000,
+        "type": "sds",
+        "lb_type": "round_robin",
+        "ssl_context": {
+          "cert_chain_file": "/etc/envoyauth/cert_chain",
+          "private_key_file": "/etc/envoyauth/private_key",
+          "ca_cert_file": "/etc/envoyauth/ca_cert",
+          "verify_subject_alt_name": [
+            "istio:accountA.namespace1.foo.com",
+            "istio:accountB.namespace1.foo.com"
```

Comment:
We need to add RDS and CDS response for HTTP routes and clusters. The config only contains TCP routes. Are we terminating TLS for TCP as well?